src/nvim/eval.c

```diff
@@ -7287,8 +7287,7 @@ static void f_browse(typval_T *argvars, typval_T *rettv)
  */
 static void f_browsedir(typval_T *argvars, typval_T *rettv)
 {
-  rettv->vval.v_string = NULL;
-  rettv->v_type = VAR_STRING;
+  f_browse(argvars, rettv);
```

Comment:
This is weird. Turns out we removed `FEAT_BROWSE`. I think we should actually restore that. https://github.com/neovim/neovim/issues/2160